scripts/ci/docker-compose/backend-mssql-bind-volume.yml

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 ---
-version: "2.2"
+version: "3.7"
```

Comment:
This field is deprecated according to compose-spec. We should delete this field if it causes any unexpected problems. https://github.com/compose-spec/compose-spec/blob/master/spec.md#version-top-level-element

Reply:
Ah. Interesting Will delete.

Reply:
Deleted. Seems no side effects. Good catch. TIL